src/components/PostSelector/PostSelector.js

```diff
@@ -0,0 +1,236 @@
+import { isEmpty } from 'lodash'
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import useInView from 'react-cool-inview'
+import { useDrag, useDrop } from 'react-dnd'
+import { useDispatch, useSelector } from 'react-redux'
+import update from 'immutability-helper'
+import { FETCH_POSTS } from 'store/constants'
+import Icon from 'components/Icon'
+import Loading from 'components/Loading'
+import RoundImage from 'components/RoundImage'
+import useDebounce from 'hooks/useDebounce'
+import { fetchPosts } from 'routes/Stream/Stream.store'
+import isPendingFor from 'store/selectors/isPendingFor'
+
+import './PostSelector.scss'
+
+const PAGE_SIZE = 10
+
+export default function PostSelector ({ collection, draggable, group, onRemovePost, onReorderPost, onSelectPost, posts }) {
+  const dispatch = useDispatch()
+  const [autocomplete, setAutocomplete] = useState('')
+  const [suggestions, setSuggestions] = useState([])
+  const [offset, setOffset] = useState('')
+  const [selectedPosts, setSelectedPosts] = useState(posts || [])
+  const [suggestionsOpen, setSuggestionsOpen] = useState(false)
+  const searchBox = useRef(false)
```

Comment:
Why `false`? Does it set the value of `current` to `false` instead of `undefined` ? Sincerely curious, I don't know the answer.

Reply:
yeah it does, and i dont know why, i think i just copied this from somewhere else